explorer/src/components/account/BlockHistoryCard.tsx

```diff
@@ -0,0 +1,112 @@
+import React from "react";
+import { TableCardBody } from "components/common/TableCardBody";
+import { useBlock, useFetchBlock } from "providers/accounts/block";
+import { Signature } from "components/common/Signature";
+
+export function BlockHistoryCard({
+  slot,
+  blockhash,
+}: {
+  slot: string;
+  blockhash: string;
+}) {
+  const block = useBlock(slot ? slot : blockhash);
+  const fetchBlock = useFetchBlock();
+  const refresh = () =>
+    fetchBlock(slot ? slot : blockhash, slot ? "slot" : "blockhash");
+
+  React.useEffect(() => {
+    if (!block) refresh();
+  }, [slot, blockhash, block]); // eslint-disable-line react-hooks/exhaustive-deps
+
+  function invalidOverview() {
+    return (
+      <tr>
+        <td>ERROR</td>
+        <td>Could not retrieve block details</td>
+      </tr>
+    );
+  }
+
+  function invalidTransactions() {
+    return (
+      <tr>
+        <td>ERROR</td>
+        <td>Could not retrieve transactions</td>
+      </tr>
+    );
+  }
```

Comment:
Please match the error state for accounts / transactions. Looks like this for invalid value

![Screen Shot 2020-10-09 at 2 16 03 PM](https://user-images.githubusercontent.com/1076145/95549580-2203fe00-0a3a-11eb-8bd4-e87686de9561.png)

Looks like this when not found
![Screen Shot 2020-10-09 at 2 18 32 PM](https://user-images.githubusercontent.com/1076145/95549702-52e43300-0a3a-11eb-8f92-c9fa26cfd079.png)